test/Driver/cr_unit_tests.jl

```diff
@@ -0,0 +1,143 @@
+using CLIMA
+using CLIMA.Atmos
+using CLIMA.ConfigTypes
+using CLIMA.PlanetParameters
+using CLIMA.MoistThermodynamics
+using CLIMA.PlanetParameters
+using CLIMA.VariableTemplates
+using CLIMA.Grids
+using CLIMA.ODESolvers
+
+using MPI
+using Printf
+using StaticArrays
+using Test
+
+Base.@kwdef struct AcousticWaveSetup{FT}
+    domain_height::FT = 10e3
+    T_ref::FT = 300
+    α::FT = 3
+    γ::FT = 100
+    nv::Int = 1
+end
+
+function (setup::AcousticWaveSetup)(bl, state, aux, coords, t)
+    # callable to set initial conditions
+    FT = eltype(state)
+
+    λ = longitude(bl.orientation, aux)
+    φ = latitude(bl.orientation, aux)
+    z = altitude(bl.orientation, aux)
+
+    β = min(FT(1), setup.α * acos(cos(φ) * cos(λ)))
+    f = (1 + cos(FT(π) * β)) / 2
+    g = sin(setup.nv * FT(π) * z / setup.domain_height)
+    Δp = setup.γ * f * g
+    p = aux.ref_state.p + Δp
+
+    ts = PhaseDry_given_pT(p, setup.T_ref)
```

Comment:
```suggestion
    ts = PhaseDry_given_pT(p, setup.T_ref, param_set)
```

Reply:
Using `param_set` here causes an error on the GPU. Is that expected? Does that mean this must run on CPU then?

Reply:
It's fine to leave this out for now, then